enterprise/frontend/src/embedding-sdk/hooks/private/use-init-data.ts

```diff
@@ -25,18 +23,38 @@ interface InitDataLoaderParameters {
 export const useInitData = ({ config }: InitDataLoaderParameters) => {
   const { allowConsoleLog = true } = config;
 
+  // react calls some lifecycle hooks twice in dev mode, the auth init fires some http requests and when it's called twice,
```

Comment:
I spent a lot of time debugging some iterations of this PR, because getOrRefresh token returns null when loading, when the init thunk was dispatched twice all sort of issues came up